domain-models-runtime/src/main/resources/templates/db_scripts/general_functions.ftl

```diff
@@ -110,3 +110,8 @@ BEGIN
   RETURN NEW;
 END;
 $$ language 'plpgsql';
+
+
+create or replace function concat_space_sql(VARIADIC text[])
+RETURNS text AS $$ select concat_ws(' ', VARIADIC $1); 
+$$ LANGUAGE SQL IMMUTABLE;
```

Comment:
Can you explain on this pull request why concat_space_sql has been choosen in favor of text_concat_ws?

Reply:
The choice of name here is based on the need for an immutable function to use in indexes.  The function is native sql and defines the concat character as space.